cli/create/codegen/generate_mode.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/dave/jennifer/jen"
+)
+
+func generateFileModeFile(path string, filename string) error {
+	f := jen.NewFile("static")
+	f.Comment("This file is generated! DO NOT EDIT\n")
+
+	fileModeMap, err := getFileModes(path)
+
+	if err != nil {
+		return err
+	}
+
+	f.Var().Id("FileModes").Op("=").Map(jen.String()).Int().Values(jen.DictFunc(func(d jen.Dict) {
+		for key, element := range fileModeMap {
+			d[jen.Lit(key)] = jen.Lit(element)
```

Comment:
Result file will contain decimal values (e.g. `493` for `0755`). I think it would be helpful in the future to add comments with filemodes in octal format.